### PR TITLE
Updated deprectaed APIs

### PR DESCRIPTION
**Pull request by nikhilTechRacers**
Updated deprecated APIs
Added deposit address service
mentions #67

### CODE REVIEW

**Comment by nikhilTechRacers** [comment]
@adshao

**Comment by adshao** [comment]
@ndashore test failed: https://travis-ci.org/adshao/go-binance/builds/453944604?utm_source=github_status&utm_medium=notification

**Comment by nikhilTechRacers** [comment]
@adshao All tests passed.

File: README.md [hunk 1]
```diff
@@ -41,8 +41,9 @@ Init client for API services. Get APIKey/SecretKey from your binance account.
 var (
     apiKey = "your api key"
     secretKey = "your secret key"
+    baseURL = "https://www.binance.com" // Can use your own mock environment for testing
```
Comment (adshao) [inline README.md]: there is no need to move baseURL to NewClient, you can update it simply by `client.BaseURL = xxx`

File: deposit_service.go [hunk 2]
```diff
@@ -83,6 +83,48 @@ type Deposit struct {
...
 	Asset      string  `json:"asset"`
-	Status     int     `json:"status"`
+	Address    string  `json:"address"`
+	AddressTag string  `json:"addressTag"`
 	TxID       string  `json:"txId"`
+	Status     int     `json:"status"`
+}
+
+// GetDepositAddressService get deposit address
+type GetDepositAddressService struct {
```
Comment (adshao) [inline deposit_service.go]: Any unit tests for this service?

**Comment by adshao** [comment]
Hi @nikhilTechRacers , please check the review comments

**Pull request closed without merging**